### PR TITLE
added provisioning dir

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

**Review by bgalek — CHANGES_REQUESTED** [review]
please remove /dist/folder from PR :)

**Comment by ar4s** [comment]
> @ar4s imo this will create double timestamp (it's artifactory feature)
I tested that and looks fine :) I was strongly inspired by what is in the tycho.

**Comment by bgalek** [comment]
@ar4s ok, LGTM then! if you tested that version, I'll can release new version ;)

File: utils/provisioning.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const fetch = require('node-fetch');
+
+const { compressDirectory } = require('./compress');
+const { provisioningArtifactUrl } = require('./artifactory');
+
+module.exports = {
+  publishProvisioning: (username, password, host, group, name, version, currentBranch, isSnapshot, tychoPath) => {
+    const provisioningDir = 'provisioning';
+    if (!fs.existsSync(provisioningDir)) {
+      fs.mkdirSync(provisioningDir);
```
Comment (bgalek) [inline utils/provisioning.js:13]: this is a problem for multimodule applications ;(
https://github.com/allegro-internal/opbox-core/tree/master/core/provisioning

**Pull request merged**